notebook/static/notebook/js/notebook.js

```diff
@@ -2055,11 +2082,11 @@ define(function (require) {
      * @param {integer} end - index of the last cell to execute (exclusive)
      */
     Notebook.prototype.execute_cell_range = function (start, end) {
-        this.command_mode();
```

Comment:
I didn't truly remove this -- this method uses `execute_cells` now, which switches into command mode.